common/src/main/java/org/opensearch/ml/common/transport/memorycontainer/memory/MLDeleteWorkingMemoryAction.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.memorycontainer.memory;
+
+import org.opensearch.action.ActionType;
+import org.opensearch.action.delete.DeleteResponse;
+
+public class MLDeleteWorkingMemoryAction extends ActionType<DeleteResponse> {
+    public static final MLDeleteWorkingMemoryAction INSTANCE = new MLDeleteWorkingMemoryAction();
+    public static final String NAME = "cluster:admin/opensearch/ml/memory_containers/memory/working/delete";
```

Comment:
Not urgent, we could discuss and fix in next PR. we may want to keep it consistent with the REST API 
```suggestion
    public static final String NAME = "cluster:admin/opensearch/ml/memory_containers/memories/working/delete";
```

Reply:
Check with connector or model, this place is singular.